client/ayon_blender/plugins/publish/extract_fbx.py

```diff
@@ -55,20 +56,22 @@ def process(self, instance):
                 new_materials.append(mat)
                 new_materials_objs.append(obj)
 
-        scale_length = bpy.context.scene.unit_settings.scale_length
-        bpy.context.scene.unit_settings.scale_length = 0.01
-
-        with bpy.context.temp_override(**context):
-            # We export the fbx
-            bpy.ops.export_scene.fbx(
-                filepath=filepath,
-                use_active_collection=False,
-                use_selection=True,
-                mesh_smooth_type='FACE',
-                add_leaf_bones=False
-            )
-
-        bpy.context.scene.unit_settings.scale_length = scale_length
+        attr_values = self.get_attr_values_from_data(instance.data)
+        scene_overrides = {}
+        unit_scale = attr_values.get("unitScale")
+        if unit_scale is not None:
+            scene_overrides["unit_settings.scale_length"] = unit_scale
+
+        with lib.attribute_overrides(bpy.context.scene, scene_overrides):
+            with bpy.context.temp_override(**context):
+                # We export the fbx
+                bpy.ops.export_scene.fbx(
+                    filepath=filepath,
+                    use_active_collection=False,
+                    use_selection=True,
+                    mesh_smooth_type='FACE',
+                    add_leaf_bones=False
+                )
```

Comment:
```suggestion
        with bpy.context.temp_override(**context):
            # We export the fbx
            bpy.ops.export_scene.fbx(
                filepath=filepath,
                use_active_collection=False,
                use_selection=True,
                mesh_smooth_type='FACE',
                add_leaf_bones=False
            )
```

Reply:
I find out that it ends up with some frame range issue when loaded back to maya. We still need to have scene_overrides for storing the frame range data.
```suggestion
        scene_overrides = {
            "frame_start": instance.data.get("frameStart"),
            "frame_end": instance.data.get("frameEnd"),
            "frame_step": instance.data.get("frameStep"),
            "render.fps": instance.data.get("fps")
        }
        # Skip None value overrides
        scene_overrides = {
            key: value for key, value in scene_overrides.items()
            if value is not None
        }
        if "render.fps" in scene_overrides:
            scene_overrides["render.fps_base"] = 1

        with lib.attribute_overrides(bpy.context.scene, scene_overrides):
            with bpy.context.temp_override(**context):
```